atd-vze/src/views/Crashes/CR3Record.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Card, CardHeader, CardBody, Button, Alert } from "reactstrap";
+import axios from "axios";
+
+function CR3Record(props) {
+  const requestCR3 = () => {
+    console.log(props);
```

Comment:
There might be a good reason for this but just in case.

Reply:
nope